core/src/main/scala/org/apache/spark/scheduler/CoarseCookSchedulerBackend.scala

```diff
@@ -193,8 +193,16 @@ class CoarseCookSchedulerBackend(
   private[this] val mesosSchedulerBackend =
     new MesosCoarseGrainedSchedulerBackend(scheduler, sc, "", sc.env.securityManager)
 
-  override def sufficientResourcesRegistered(): Boolean =
-    totalExecutorsAcquired >= executorLimit * schedulerContext.minRegisteredResourceRatio
```

Comment:
Can we change the loop code that prints out this number and replace it with the getExecutorIds().lenght?

Also, can we rename totalExecutorsAcquired to "totalOutstandingRequestedExecutors" or something?